### PR TITLE
Uploading Ironbit Task

### PR DESCRIPTION
**Pull request by manuelEnriqueIronbit**
Add unit testing exercise  and inheritance task, remove consoles.log from dataStructures file

### CODE REVIEW

File: manuelEnriqueIronbit/practica-2019-12-09/umbrellaTesting/src/function-test/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+function minUmbrellas(weather) {
```
Comment (alfonsorios96) [inline manuelEnriqueIronbit/practica-2019-12-09/umbrellaTesting/src/function-test/index.js]: ES5 not allowed, change to Arrow function

File: manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+class Empleado {
```
Comment (alfonsorios96) [inline manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js]: Not spanish

File: manuelEnriqueIronbit/practica-2019-12-09/alphabetPosition/src/function-test/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      let letter = String.fromCharCode(i+97);
+      arrayLow[i]=letter;
+  }
+
+  //MAPEANDO EL ARREGLO DE MINUSCULAS
+  //DECONSTRUYENDO EL ARRAY EN CARACTERES
+  let phraseArray = Array.from(text);
+  //IGNORANDO LOS ESPACIOS DEL ARREGLO 
+  let ignoreSpaces = phraseArray.filter(c=>c!=' ');
+  console.log(ignoreSpaces);
```
Comment (alfonsorios96) [inline manuelEnriqueIronbit/practica-2019-12-09/alphabetPosition/src/function-test/index.js]: Remove `console.log`

File: manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
+class Employee {
+    constructor(totalHrs, workingHrs, freeHrs ){
+        this.totalHours = totalHrs;
+        this.workingHours = workingHrs;
+        this.freeHours = freeHrs;
+        this.schedule = [];
+    }
+    assignHours(){
```
Comment (alfonsorios96) [inline manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js]: remove unused methods

File: manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        super(totalHrs, workingHrs, freeHrs);
+        this.schedule = [
+            [null, null, null, null, null, null, null, null, null],
+            [null, null, null, null, null, null, null, null, null],
+            [null, null, null, null, null, null, null, null, null],
+            [null, null, null, null, null, null, null, null, null],
+            [null, null, null, null, null, null, null, null, null]
+        ] 
+    }
+    /*assignHours(){
```
Comment (alfonsorios96) [inline manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js]: Remove unused code

File: manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js (hunk 2 above)
Comment (manuelEnriqueIronbit) [reply manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js]: removed the file from master branch

File: manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js (hunk 5 above)
Comment (manuelEnriqueIronbit) [reply manuelEnriqueIronbit/practica-2019-12-10/inheritance/schedule.js]: Removed filed because not finished

**Pull request merged**